Classes/MOC/Varnish/Aspects/ContentCacheAspect.php

```diff
@@ -32,17 +41,19 @@ class ContentCacheAspect {
 	 * @param JoinPointInterface $joinPoint
```

Comment:
instead of injecting the settings and changing the methods, use the AOP setting condition like
````
@Flow\AfterReturning(setting(MOC.Varnish.enabled) && method(TYPO3\TypoScript\Core\Cache\RuntimeContentCache->postProcess())")
````

see http://flowframework.readthedocs.io/en/stable/TheDefinitiveGuide/PartIII/AspectOrientedProgramming.html#setting

same for the other ones..